.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Veracode Upload And Scan
         uses: veracode/veracode-uploadandscan-action@0.2.2
         with:
-          appname: 'Mix.cli'
+          appname: 'mix-cli'
```

Comment:
Please revert this change here and the one in .github/workflows/veracode.yaml. mix-cli is known as "Mix.cli" to Veracode and we cannot change that. This is not something users see so that's good.